lib/src/voip/voip.dart

```diff
@@ -573,87 +647,118 @@ class VoIP {
 
   /// Make a P2P call to room
   ///
-  /// [roomId] The room id to call
+  /// Pretty important to set the userId, or all the users in the room get a call.
+  /// Including your own other devices
+  ///
+  /// A userId is mandatory to if you want to set a deviceId
+  /// Setting the deviceId would make all other devices for that userId ignore the call
+  /// Ideally only group calls would need setting both userId and deviceId to allow
+  /// having 2 devices from the same user in a group call
+  ///
+  /// For p2p call, you want to have all the devices of the specified `userId` ring
   ///
-  /// [type] The type of call to be made.
-  Future<CallSession> inviteToCall(String roomId, CallType type) async {
+  /// yes I forced you to call a particular user in a room, letting anyone answer
+  /// doesn't make sense
+  Future<CallSession> inviteToCall(
```

Comment:
Should we just require passing the room here?

Reply:
well it's just `final room = client.getRoomById(roomId);` either in the client or in the sdk. idm either way

Reply:
I would then suggest requiring to pass the room everywhere for consistency. Especially since we seem to be getting the room already just to get the id and then the room from the id again....

Reply:
done